### PR TITLE
visualization blog edits

### PR DESCRIPTION
**Pull request by jspaaks**
This PR adds a couple of edits on the visualization blog, mostly style and rephrase and such.

### CODE REVIEW

File: show-me-the-visualization/cli.cpp [hunk 1]
```diff
@@ -4,8 +4,8 @@
 #include "newtonraphson.hpp"
 
 int main() {
-  double initial_guess = -4;
-  double tolerance = 0.001;
+  float initial_guess = -4;
```
Comment (sverhoeven) [inline show-me-the-visualization/cli.cpp:7]: Nice inconsistency find

File: show-me-the-visualization/README.md [hunk 2]
```diff
@@ -119,14 +119,14 @@ index = 4 x = -1.02 y = -0.28 slope = 14.40 delta_x = -0.02
...
 
-The last iteration has `-1.00` as x, which is what we expected.
+The last iteration has `x = -1.00`, which is what we expected.
 
 ## Bindings
 
-Emscripten can handle simple types like float and int, but needs help exposing more complex types to JavaScript like the iterations property.
-We need to use [value_object](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#value-types) to expose the Iteration struct and [register_vector](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#built-in-type-conversions) as the iterations property type.
+Emscripten can handle simple types like `float` and `int`, but needs help exposing more complex types to JavaScript like the `iterations` property.
+We need to use [`value_object`](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#value-types) to expose the `Iteration` `struct` and [`register_vector`](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#built-in-type-conversions) as the `iterations` property type.
```
Comment (sverhoeven) [inline show-me-the-visualization/README.md:127]: Links are already rendered differently, they do not need more highlighting
```suggestion
We need to use [value_object](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#value-types) to expose the `Iteration` `struct` and [register_vector](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#built-in-type-conversions) as the `iterations` property type.
```

File: show-me-the-visualization/README.md [hunk 3]
```diff
@@ -154,41 +154,23 @@ EMSCRIPTEN_BINDINGS(newtonraphson) {
...
 
-Same as in [previous blog](TODO) we can compile to a WebAssembly module with Emscripten using `emcc` command
+We can now compile our C++ code to a WebAssembly module with Emscripten using `emcc` command, exactly [like we did before](../run-your-c%2B%2B-code-on-the-web):
 
 ```shell
 emcc -I. -o newtonraphson.js -Oz -s MODULARIZE=1 \
   -s EXPORT_NAME=createModule --bind newtonraphson.cpp bindings.cpp
 ```
 
-<!--
```
Comment (sverhoeven) [inline show-me-the-visualization/README.md:164]: Ah, I forgot about that one, nice to see #65

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Looks good to me. Sentences are smoother now.

Lots more quotes, some sentences now have a bit to many quotes for me. Feels like every odd word is quoted. Especially line 22, 50, 126, 127

Can you correct the links on the plot images so they point to renamed directory?

File: show-me-the-visualization/README.md (hunk 2 above)
Comment (jspaaks) [reply show-me-the-visualization/README.md:127]: strictly speaking I don't care to highlight them, but I want to emphasize that I'm referring to something in the code. It's your call though.

**Comment by jspaaks** [comment]
> Lots more quotes, some sentences now have a bit to many quotes for me. Feels like every odd word is quoted. Especially line 22, 50, 126, 127

Agree, but not quoting them is not the right solution to me

**Review by sverhoeven — APPROVED** [review]
I can not think of another way to tell the string is part of a code snippet.
Lets keep the quotes for consistencies sake.

**Pull request merged**